### PR TITLE
docs(readme): update the contributors list

### PR DESCRIPTION
**Pull request by Berkmann18**
Updated with what `node ../src/cli.js -t $GITHUB_TOKEN -r name-your-contributors -u mntnr --commits | jq '[ .[][].login | " - [@\(.)](https://github.com/\(.))"]' | jq unique | jq -r .[]` returned even tho the initial list wasn't __alphabetically sorted__.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -48,16 +48,19 @@ It spits out to a JSON object, because that's the most portable system for using
 
 These people have helped make Name Your Contributors:
 
- - [@diasdavid](https://github.com/diasdavid)
+ - [@Berkmann18](https://github.com/Berkmann18)
+ - [@RichardLitt](https://github.com/RichardLitt)
```
Comment (Berkmann18) [inline readme.md]: Just noticed that all but this line were in alphabetical order, I wonder why?

Comment (RichardLitt) [reply readme.md]: Should we sort alphabetically, anyway?

Comment (RichardLitt) [reply readme.md]: Really, we shouldn't have to rely on JQ for this.

Comment (Berkmann18) [reply readme.md]: I don't think we should sort that way (other than by contributions) but that was what I noticed at first.
But I was playing around with `jq` and noticed it was the same kind of output but the README's one didn't seem to be up-to-date.

Comment (RichardLitt) [reply readme.md]: Looks like we need a bot. :D

Comment (Berkmann18) [reply readme.md]: That would be a good idea 😄.

**Pull request merged**